experiment/resultstore/convert.go

```diff
@@ -124,6 +132,32 @@ func convert(project, details string, url gcs.Path, result downloadResult) (resu
 				URL:         buildLog, // ensure build-log.txt appears as the invocation log
 			},
 		},
+		Properties: []resultstore.Property{
+			{
+				Key:   "Job Name",
+				Value: jobName,
+			},
+			{
+				Key:   "Started Timestamp",
+				Value: strconv.FormatInt(started.Timestamp, 10),
+			},
+			{
+				Key:   "Node",
+				Value: started.Node,
+			},
+			{
+				Key:   "Pull",
+				Value: started.Pull,
+			},
+		},
+	}
+
+	for k, v := range started.Repos {
+		property := resultstore.Property{
+			Key:   "Repo",
+			Value: fmt.Sprintf("%s:%s", k, v),
```

Comment:
Is there any ability to be more structured? Otherwise someone's just going to have to write string manipulation on the other end, too

Reply:
The resultstore property metadata is just a string field so we couldn't really introduce structure here. However, this is more of a label used for searching/identification purpose, and it outputs repo:branch, which is the same format as the started.json.

Reply:
Can you choose better names than k and v? What is v?

Reply:
Done, k = repo and v = branch